Source/FileManager/BackgroundFileSystem.cs

```diff
@@ -134,6 +134,7 @@ private void RemovePath(string path)
 
         private void AddPath(string path)
         {
+            if (!File.Exists(path)) return;
```

Comment:
SaveFileDialog does something weird.  It triggers the scanner to think there's a file, but there isn't one.

Reply:
Good catch. That is indeed weird